cookbook/form/form_customization.rst

```diff
@@ -114,6 +114,13 @@ fragment needed to render every part of a form:
   but the CSS classes applied are the ones used to display the forms horizontally
   (i.e. the label and the widget in the same row).
 
+.. caution::
+
+  If you use the Bootstrap form theme and want to render the parts of 
+  a checkbox field individually, do not use the Twig function {{ form_label() }}. 
```

Comment:
You should put the code between double backticks, to make it an inline code block.